packages/uiweb/src/lib/context/chatAndNotification/chatAndNotificationMainContext.tsx

```diff
@@ -0,0 +1,47 @@
+import { createContext, useState } from 'react';
+import type { PushSubTabs, PushTabs } from '../../types';
+import { PUSH_TABS } from '../../types';
+
+export type MainContextType = {
```

Comment:
Rename the context as ChatAndNotificationMainContextType and ChatAndNotificationMainContext